config/checkstyle/suppressions.xml

```diff
@@ -1,32 +1,11 @@
 <?xml version="1.0"?>
 
-<!--
-  ~ Copyright 2015 Vincent Brison.
-  ~
-  ~ Licensed under the Apache License, Version 2.0 (the "License");
-  ~ you may not use this file except in compliance with the License.
-  ~ You may obtain a copy of the License at
-  ~
-  ~    http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing, software
-  ~ distributed under the License is distributed on an "AS IS" BASIS,
-  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  ~ See the License for the specific language governing permissions and
-  ~ limitations under the License.
-  -->
+
 
 <!DOCTYPE suppressions PUBLIC
     "-//Puppy Crawl//DTD Suppressions 1.1//EN"
     "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
 
 <suppressions>
-    <suppress checks="[a-zA-Z0-9]*" files="R.java" />
```

Comment:
@weijjia I removed all of these suppressions because they appear to have been copy and pasted from someone else's file.  I will add them back in as necessary, but for now I am not seeing any warnings for R files.